website/docs/Pipelines/RNA_with_UMIs_Pipeline/README.md

```diff
@@ -24,24 +24,37 @@ While this workflow was created to be used with TCap RNA-seq data, it can be use
 Check out the [RNA with UMIs Methods](./rna-with-umis.methods.md) section to get started!
 :::
 
-<!--- quickstart table will go here --->
+## Quickstart table
+
+| Pipeline features | Description | Source | 
+| --- | --- | --- |
+| Assay type | TCap (or any bulk) RNA-seq data | [Cieslik et al. 2015](https://genome.cshlp.org/content/25/9/1372) |
+| Overall workflow | Read alignment and transcriptome quantification | Code available from [GitHub](https://github.com/broadinstitute/warp/blob/master/pipelines/broad/rna_seq/RNAWithUMIsPipeline.wdl) |
+| Workflow language | WDL 1.0 | [openWDL](https://github.com/openwdl/wdl) |
+| Genomic reference sequence | GRCh38 (hg38) and GRCh37 (hg19) human genome primary sequence | Genome Reference Consortium [GRCh38](https://www.ncbi.nlm.nih.gov/assembly/GCF_000001405.39) and [GRCh37](https://www.ncbi.nlm.nih.gov/assembly/GCF_000001405.25/) |
```

Comment:
@kayleemathews Is this actually where they got the reference? Just confirming because it's different from our other pipelines.

Reply:
Yeah, Takuto told me we are using the GRC reference instead of the GENCODE one.